rules/do-you-use-voice-recordings-when-appropriate/rule.md

```diff
@@ -1,33 +1,49 @@
 ---
-seoDescription: Do you use videos or voice recordings instead of sending long emails?
 type: rule
-archivedreason:
-title: Communication - Do you use videos or voice recordings instead of sending long emails?
-guid: 5c824e64-1491-4957-baa2-3e18bdd00299
+title: Communication - Do you use videos or voice recordings instead of sending
+  long emails?
+seoDescription: Do you use videos or voice recordings instead of sending long emails?
 uri: do-you-use-voice-recordings-when-appropriate
-created: 2014-02-25T15:46:04.0000000Z
 authors:
   - title: Adam Cogan
     url: https://ssw.com.au/people/adam-cogan
 related: []
 redirects:
   - communication-do-you-use-videos-or-voice-recordings-instead-of-sending-long-emails
+created: 2014-02-25T15:46:04.000Z
+archivedreason: null
+guid: 5c824e64-1491-4957-baa2-3e18bdd00299
 ---
-
-Sometimes it is quicker to explain something than it is to write a comprehensive email. This is particularly true when explaining an idea or concept relating to a product, and you need to go backwards and forwards many times to clarify details. If you can't easily speak directly with someone, due to schedule, travel, or time zone constraints try sending a voice recording instead.
+Sometimes it is quicker to explain something verbally than it is to write a comprehensive email. This is particularly true when explaining an idea or concept relating to a product, and you need to go backwards and forwards many times to clarify details. If you can't easily speak directly with someone, due to schedule, travel, or time zone constraints try sending a voice recording instead.
 
 <!--endintro-->
 
 Sending videos or voice recordings can be thought of as a slow paced conversation. There are many benefits to sending voice messages instead of email when iterating over concepts and ideas.
 
-Sending and listening to voice recordings feels more personal than reading an email, because it is possible to hear the person's mood and emphasis more naturally. It is also helpful if the recipient needs to re-play a portion of the message to understand it, especially for people who are communicating in a language other than their native tongue.
+**Advantages of Using Videos or Voice Recordings:**
+
+1. Personal Touch: They convey emotion and emphasis more naturally, making communication feel more personal.
```

Comment:
Hi @Pam623 

These lists would look better if the prefix is bolded, the separator is a dash instead of a colon, and [no full stops](https://www.ssw.com.au/rules/avoid-full-stops/)

from

> 1. Personal Touch: They convey emotion and emphasis more naturally, making communication feel more personal.

To:

> 1. **Personal Touch** - They convey emotion and emphasis more naturally, making communication feel more personal

- [ ] 1. Please apply to all list items in the rule